### DISCUSSION TITLE
Small typo

### DISCUSSION THREAD

**Discussion by fotoente** [yupix/Mi.py#58] [report bug] [answered]
```if __name__ == '__main__':
    uri = "wss://example.com/streaming"
    token = "This is your token"

    bot = MyBot('!')
    asyncio.run(bot.start(url, token))
```

`uri` in the first line, should be `url`

**Comment by yupix** [comment, marked as answer]
I'm not very familiar with this, but I've seen in the past that uri is made up of url + urn, so in a sense it's not a typo, but it seems to be a lesser recognized term. I also feel it, so I may replace it in the near future

https://stackoverflow.com/questions/176264/what-is-the-difference-between-a-uri-a-url-and-a-urn
